scripts/install

```diff
@@ -9,164 +9,90 @@
-#=================================================
-# DOWNLOAD, CHECK AND UNPACK SOURCE
-#=================================================
-ynh_script_progression --message="Setting up source files..." --weight=5
-
-ynh_app_setting_set --app=$app --key=final_path --value=$final_path
+# Create a dedicated PHP-FPM configy
+ynh_add_fpm_config --usage=low --footprint=low
 
-# 1 - Clone stable repo
-git clone --quiet https://github.com/friendica/friendica.git -b stable "$final_path"
-# Reset branch to the level of update we needed
-pushd "$final_path"
-	git reset --hard --quiet $version_commit
-popd
+# Create a dedicated nginx config
+ynh_add_nginx_config
 
-# 2 - Clone addons repo
-git clone --quiet https://github.com/friendica/friendica-addons.git -b stable "$final_path/addon"
-# Reset addons branch to the level of update we needed
-pushd "$final_path/addon"
-	git reset --hard --quiet $addons_version_commit
-popd
+# Create a dedicated systemd config
+ynh_add_systemd_config
 
-chmod 750 "$final_path"
-chmod -R o-rwx "$final_path"
-chown -R $app:www-data "$final_path"
+yunohost service add $app --description="Friendica daemon" --log="/var/log/$app/$app.log"
 
-#=================================================
-# NGINX CONFIGURATION
-#=================================================
-ynh_script_progression --message="Configuring NGINX web server..." --weight=1
+# Use logrotate to manage application logfile(s)
+ynh_use_logrotate
 
-# Create a dedicated nginx config
-ynh_add_nginx_config
+# Create a dedicated Fail2Ban config
+ynh_add_fail2ban_config --logpath="/var/log/nginx/${domain}-error.log" --failregex="^.*authenticate\: failed login attempt.*\"ip\"\:\"<HOST>\".*$"
 
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
 ynh_script_progression --message="Adding the LDAP Auth addon configuration file..." --weight=1
 
 # LDAP addon config
-ynh_add_config --template="../conf/addon.config.php" --destination="$final_path/config/addon.config.php"
-
-#=================================================
-# PHP-FPM CONFIGURATION
-#=================================================
-ynh_script_progression --message="Configuring PHP_FPM..." --weight=4
-
-# Create a dedicated PHP-FPM configy
-ynh_add_fpm_config
-phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
+ynh_add_config --template="../conf/addon.config.php" --destination="$install_dir/config/addon.config.php"
 
 # 3 - some extra folders
-mkdir -p "$final_path/view/smarty3"
-chmod -R 775 "$final_path/view/smarty3"
-
-#=================================================
-# SETUP A CRON
-#=================================================
-ynh_script_progression --message="Setting up the cron job..." --weight=1
-
-ynh_add_config --template="../conf/cron" --destination="/etc/cron.d/$app"
-chown root: "/etc/cron.d/$app"
-chmod 644 "/etc/cron.d/$app"
+mkdir -p "$install_dir/view/smarty3"
+chmod -R 775 "$install_dir/view/smarty3"
```

Comment:
This shouldn't be needed if the webserver/application user already owns the base install folder and the whole application file tree like you declared in `scripts/upgrade`:
https://github.com/YunoHost-Apps/friendica_ynh/blob/7e2813cfb8399be8c4b648dbb97876f2cdd0f633/scripts/upgrade#L48-L49